### PR TITLE
Redirect to seamonkey/thunderbird based on user agent

### PR DESCRIPTION
**Pull request by kumar303**
Fixes https://github.com/mozilla/addons-frontend/issues/3728

### CODE REVIEW

**Comment by kumar303** [comment]
@muffinresearch should we cherry pick this?

File: config/default.js [hunk 1]
```diff
@@ -222,6 +222,8 @@ module.exports = {
   validClientApplications: [
     'android',
     'firefox',
+    'seamonkey',
```
Comment (tofumatt) [inline config/default.js]: Will this allow the frontend to navigate to seamonkey and thunderbird URLs? Do we want that?

**Comment by kumar303** [comment]
It sounds like you answered the questions but here are the main points:

* After this patch you can browse the site with either of these client apps in the URL and nothing blows up. You can only browse the site like this on your local dev server though.
* On dev, stage, and prod the nginx rules will serve these client app URLs from `addons-server`. However, without this patch it won't redirect correctly for the user agents.

**Comment by muffinresearch** [comment]
We shouldn't need this patch see my comments in #3728

**Pull request closed without merging**